save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt

```diff
@@ -37,4 +49,17 @@ class TestExecutionService(private val testExecutionRepository: TestExecutionRep
         }
         return lostTests
     }
+
+    fun saveTestExecution(testDtos: List<TestDto>) {
+        testDtos.map { testDto ->
+            testRepository.findByHash(testDto.hash).ifPresentOrElse(
+                { test ->
+                    testSuiteRepository.findById(testDto.testSuiteId).ifPresent {
+                        testExecutionRepository.save(TestExecution(test, testDto.testSuiteId, null, it.project!!, TestResultStatus.READY, null, null))
```

Comment:
We don't actualy need a `Project` field in `TestExecution`, I think it's a bug after refactorings. We already have a `Test`, that has `TestSuite`, that has `Project`. So please remove it from `TestExecution`. Then you won't need to make an additional query into testSuites table